manifests/init.pp

```diff
@@ -187,6 +189,7 @@
 # @param dumpdir
 #   Directory to store measurement history in on exit.
 class chrony (
+  Optional[String] $bindaddress                                    = undef,
```

Comment:
Does it make sense to use `Stdlib::IP::Address` here?

Also, does the service allow binding to multiple IP addresses? If so, an array of IP addresses may be more suitable (defaulting to an empty array).

Reply:
bindaddress only seems to be able to take one argument
`chronyd[10873]: Fatal error : Too many arguments for bindaddress directive at line 12 in file /etc/chrony.conf`

Agree with changing the type to Stdlib::IP::Address. @jhunt-steds are you able to update your PR?

Reply:
Thanks very much to both of you for the comments and suggestions; I'll work on updating my PR at my first opportunity, today or tomorrow (end of fiscal at my org means I'm dealing with a bunch of Red Hat licensing stuff at the moment :-\ ).

Reply:
Oh my bad, it looks like you can set multiple bindaddress addresses but it needs to be multi-line like the bindcmdaddress parameter. Should definitely replicate that logic

Reply:
Aha, I will look into that. Will definitely be tomorrow though.

By the way, what's the etiquette of resolving a conversation? Is that for the person who started it to do? Is it something people bother with at all?

Reply:
Believe its generally accepted that the author resolves the thread

Reply:
That makes sense, thanks.

I've updated it to use an array to allow all possible use cases as suggested. The [chrony.conf man page for the latest version](https://chrony.tuxfamily.org/doc/4.1/chrony.conf.html) says `bindaddress` can take at most one IPv4 and one IPv6 address; I tested this out on a throwaway server with one `bindaddress` directive on each line and chronyd does honor it. Anyway, I look forward to any more feedback you might have. Thanks for everything!

Reply:
I'm not 100% certain but I think you might have to revert the type, to match:
Array[String] $bindcmdaddress                                    = ['127.0.0.1', '::1'],

Reply:
Otherwise 💯  looks good to me!

Reply:
Do you mean I need to change the type of `bindaddress` from `Array[Stdlib::IP:Address]` back to `Array[String]` to match the type of `bindcmdaddress`? I understand that might be desirable for consistency's sake, but it passes a `bundle exec rake test` in its current form, and `Stdlib::IP::Address` accepts both IPv4 and IPv6 addresses, so there's no strictly technical reason I can see to go back to `String`. I think the tighter type is a good idea even if it's not quite the same as some things that were done previously, but if that consistency is valued more highly I am willing to switch it.

Reply:
Cool, okay it passes the tests! 👍